concepts/modularization/README.md

```diff
@@ -0,0 +1,83 @@
+# Modularization
```

Comment:
One thing missing in this concept is imho if/how we organise the source code:
Currently, all kyma components reside in the mono repo at `kyma-project/kyma`. Will this be changed?
Technically it is not necessary to split the existing code base, but for new additional components we should avoid integrating them into the existing repo. Lets describe how such component could find a way into the release channels, and how the current testing approach could be modified to include them.

Reply:
I added the section to the concept about it.